sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_patch.py

```diff
@@ -6,9 +6,48 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import Any, List, Union
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+
+from ._client import(
+    DocumentIntelligenceClient as GeneratedDocumentIntelligenceClient,
+    DocumentIntelligenceAdministrationClient as GeneratedDocumentIntelligenceAdministrationClient,
+)
+
+
+class DocumentIntelligenceClient(GeneratedDocumentIntelligenceClient):
+    __doc__ = GeneratedDocumentIntelligenceClient.__doc__
+
+    def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, TokenCredential], **kwargs: Any) -> None:
+        # The default polling interval should be 5 seconds.
+        polling_interval = kwargs.pop("polling_interval", 5)
```

Comment:
We should have some tests to make sure we are getting the correct polling interval set. We should have some tests in the old SDK as reference.

Reply:
can this just be a typed out kwarg ?

Reply:
Agree, we should make it explicitly documented and available for customers to set. For small documents like ID card, it's too long to set the default = 5s.

Reply:
@luzhang06 It is documented: https://github.com/Azure/azure-sdk-for-python/blob/ded1d82bfa8d0ea8805434843fa550402f53f491/sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_client.py#L47-L48
@johanste Do you think we should make "polling_interval" to be a typed out kwarg?

Reply:
We're adding a linter rule that requires that all document kwarg should be in the signature. That tends to say that it should be a keyword-oly argument in the signature.

Reply:
After talking with Johan, we will not make it to be a typed out kwarg, because: "As soon as the service starts returning Retry-After headers, we really don't want customers to specify this parameter. "

Reply:
Works for me, we will actually remove polling_interval from autorest docstring instead.